scripts/InstallBGInfo.ps1

```diff
@@ -0,0 +1,9 @@
+if ($null -eq (Get-Command -Name 'choco.exe' -ErrorAction SilentlyContinue)) {
+    Write-Warning "Chocolatey not installed. Cannot install BGInfo."
+}
+else {
+    choco upgrade bginfo -y --source="'c:\packages'"

```

Comment:
@pauby I think this shouldn't be pointed directly at c:\packages

Reply:
@gep13 I've removed that.